components/config/LocalPreferences.tsx

```diff
@@ -0,0 +1,91 @@
+import { ActionButton } from '@/common/buttons'
+import { Card } from '@/common/Card'
+import { Checkbox } from '@/common/forms'
+import {
+  buildGraphicPreferenceKeyForLabel,
+  GraphicMediaFilter,
+  LabelChip,
+  labelsRequiringMediaFilter,
+} from '@/common/labels'
+import { useGraphicMediaPreferences } from './useLocalPreferences'
+import { toast } from 'react-toastify'
+
+const GraphicMediaPreferenceSelectorForLabel = ({
+  label,
+}: {
+  label: string
+}) => {
+  const { getPreference } = useGraphicMediaPreferences()
+
+  return (
+    <div className="my-2">
+      <LabelChip className="ml-0 mb-2">{label}</LabelChip>
+      {(['blur', 'grayscale', 'translucent'] as GraphicMediaFilter[]).map(
```

Comment:
This list is used a few places and cast into `GraphicMediaFilter[]`.  Would it make sense to export it somewhere as a const with the proper type?

Reply:
totally, was just being lazy :(